src/cascadia/TerminalSettingsModel/KeyChordSerialization.cpp

```diff
@@ -282,5 +286,43 @@ winrt::hstring KeyChordSerialization::ToString(const KeyChord& chord)
         buffer = L"";
     }
 
-    return winrt::hstring{ buffer };
+    return buffer;
+}
+
+KeyChord KeyChordSerialization::FromString(const winrt::hstring& hstr)
+{
+    return _FromString(std::wstring{ hstr });
+}
+
+winrt::hstring KeyChordSerialization::ToString(const KeyChord& chord)
+{
+    return hstring{ _ToString(chord) };
+}
+
+KeyChord ConversionTrait<KeyChord>::FromJson(const Json::Value& json)
+{
+    const std::string keyChordText{ json.asString() };
+    try
+    {
+        return _FromString(til::u8u16(keyChordText));
+    }
+    catch (winrt::hresult_invalid_argument)
+    {
+        return nullptr;
+    }
+}
+
+bool ConversionTrait<KeyChord>::CanConvert(const Json::Value& json)
+{
+    return json.isString();
+}
+
+Json::Value ConversionTrait<KeyChord>::ToJson(const KeyChord& val)
+{
+    return til::u16u8(_ToString(val));
+}
+
+std::string ConversionTrait<KeyChord>::TypeDescription() const
+{
+    return "key chord";

```

Comment:
Huh, is there a group of this kind of string constants? Not really comfortable that a magic string just sits here.

Reply:
`ConversionTrait` comes from `JsonUtils`, where we have a ton of these raw strings going around.

That said, I don't see the harm in making it a static constexpr, so sure, I'll add it in :)

Reply:
it doesn't GET us anything. just leave it.